service/src/test/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/WorkspaceCloneUtilsTest.java

```diff
@@ -183,7 +189,8 @@ public void buildDestinationControlledGcsBucket_private_setPrivateResourceStateT
             RandomStringUtils.randomAlphabetic(5),
             "This is a cloned private bucket",
             // Gcs bucket cloud instance id must be lower-case.
-            RandomStringUtils.randomAlphabetic(5).toLowerCase(Locale.ROOT));
+            RandomStringUtils.randomAlphabetic(5).toLowerCase(Locale.ROOT),
```

Comment:
opt Why not `TestUtils.appendRandomNumber(DEFAULT_USER_EMAIL)` ? Same for rest of PR.